src/pyspextool/telluric/make_telluric_spectra.py

```diff
@@ -110,38 +115,92 @@ def make_telluric_spectra(intensity_unit:str='W m-2 um-1',
             #
             # Change units to those requested by the user
             #
-
-            flux = result[0]
-            unc = result[1]
-            vega = result[2]
             
             flux = convert_fluxdensity(standard_wavelength,
-                                       result[0],'um','erg s-1 cm-2 A-1',
+                                       result[0],
+                                       'um','erg s-1 cm-2 A-1',
                                        intensity_unit)
             
             unc = convert_fluxdensity(standard_wavelength,
-                                      result[1],'um','erg s-1 cm-2 A-1',
+                                      result[1],
+                                      'um','erg s-1 cm-2 A-1',
                                       intensity_unit)
             
             vega = convert_fluxdensity(standard_wavelength,
-                                       result[2],'um','erg s-1 cm-2 A-1',
+                                       result[2],
+                                       'um','erg s-1 cm-2 A-1',
                                        intensity_unit)
```

Comment:
Not sure where this function is defined but astropy does this: https://docs.astropy.org/en/stable/units/equivalencies.html#spectral-flux-and-luminosity-density-units

Reply:
I have tried to avoid using other packages as much as possible.  Things change, and this way if I write it, then I know what it does.  Obviously that doesn't work for everything, but for this kind of stuff I want to do it this way.

Reply:
I've opened an issue. https://github.com/pyspextool/pyspextool/issues/204